source/plugins/ruby/MdmMetricsGenerator.rb

```diff
@@ -490,6 +508,12 @@ def getContainerResourceUtilizationThresholds
           pvUsagePercentageThresholdFloat = (pvUsagePercentageThreshold.to_f).round(2)
           metric_threshold_hash[Constants::PV_USED_BYTES] = pvUsagePercentageThresholdFloat
         end
+
+        jobCompletionTimeThreshold = ENV["AZMON_ALERT_JOB_COMPLETION_TIME_THRESHOLD"]
```

Comment:
So Rashmi, i see you have made the threshold configurable, and also have kept the previous default value (6 hours), so we don't create any breaking changes, correct ?

Reply:
yes thats right. the default will still be 6 hours.